### ISSUE TITLE
Warning when running modelsummary() on modelsummary_list object.

### ISSUE THREAD

**Issue by adamaltmejd** [vincentarelbundock/modelsummary#507]
```r
url <- 'https://vincentarelbundock.github.io/Rdatasets/csv/HistData/Guerry.csv'
dat <- read.csv(url)
ml <- modelsummary(lm(Donations ~ Literacy + Clergy, data = dat), output = "modelsummary_list")
modelsummary(ml)
```

```
Objects of class 'NULL' are currently not supported.
```

**Comment by vincentarelbundock** [comment]
Thanks a lot for the report. I also just happened to run into this last night.

I'm pretty sure I know what the issue is, and it should be a very easy fix. Unfortunately, I probably won't be able to look into this for the next couple weeks, but I'll ping you as soon as it's fixed.

**Comment by vincentarelbundock** [comment]
Should be fixed by e4ad054

Please re-open or comment if it doesn't work for you.
